fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraVersionsIT.java

```diff
@@ -663,7 +662,6 @@ private static void patchLiteralProperty(final String url, final String predicat
 
     private CloseableDataset getContent(final String url) throws IOException {
         final HttpGet getVersion = new HttpGet(url);
-        getVersion.addHeader("Prefer", "return=representation; include=\"" + EMBED_CONTAINS.toString() + "\"");
```

Comment:
Instead, can we just change the header to `EMBED_CONTAINED`?

Reply:
Yes, I'll change it.